kustomize/README.md

```diff
@@ -0,0 +1,176 @@
+    ```
+    components:
+      - components/cymbal-branding
+      - components/google-cloud-operations
+      # - components/memorystore
+     ```
+
+    Note: The example above will enable both the Cymbal Branding and Google Cloud Operations components in the new deployment.
+
+    Note: To close `nano`, press `Ctrl`/`control` + `x`. To save changes, press `y`, then press `Enter`/`return`.
+
+1. Check to see what changes will be made to the existing deployment config.
+
+    ```
+    kustomize build .
+    ```
+
+1. Apply the Kustomize deployment changes to the existing deployment.
+
+    ```
+    kubectl apply -k .
+    ```
+
+    Note: It may take 2-3 minutes before the changes are reflected on the deployment.
+
+1. **Wait for the Pods to be ready.**
+
+    ```
+    kubectl get pods
+    ```
+
+    After a few minutes, you should see:
+
+    ```
+    NAME                                     READY   STATUS    RESTARTS   AGE
+    adservice-76bdd69666-ckc5j               1/1     Running   0          2m58s
+    cartservice-66d497c6b7-dp5jr             1/1     Running   0          2m59s
+    checkoutservice-666c784bd6-4jd22         1/1     Running   0          3m1s
+    currencyservice-5d5d496984-4jmd7         1/1     Running   0          2m59s
+    emailservice-667457d9d6-75jcq            1/1     Running   0          3m2s
+    frontend-6b8d69b9fb-wjqdg                1/1     Running   0          3m1s
+    loadgenerator-665b5cd444-gwqdq           1/1     Running   0          3m
+    paymentservice-68596d6dd6-bf6bv          1/1     Running   0          3m
+    productcatalogservice-557d474574-888kr   1/1     Running   0          3m
+    recommendationservice-69c56b74d4-7z8r5   1/1     Running   0          3m1s
+    shippingservice-6ccc89f8fd-v686r         1/1     Running   0          2m58s
+    ```
+
+1. **Access the web frontend in a browser** using the frontend's `EXTERNAL_IP`.
+
+    ```
+    kubectl get service frontend-external | awk '{print $4}'
+    ```
+
+    **Note**- you may see `<pending>` while GCP provisions the load balancer. If this happens, wait a few minutes and re-run the command.
+
+### Cleanup
+
+After you have run the deployment variant on Online Boutique, you will want to reset the sample application back to its vanilla state.
+
+1. While still in the `kustomize/` directory, re-apply the original Kubernetes config to the Online Boutique deployment.
+    
+    ```
+    kubectl apply -f base
+    ```
+    
+    Note: It may take 2-3 minutes before the changes are reflected on the deployment.
+
+### Clean up Memorystore
+
+If you have enabled the **Memorystore** deployment variation, complete the additional cleanup steps in the **[Additional Deployment Instructions (Memorystore)](https://github.com/GoogleCloudPlatform/microservices-demo/edit/readme/kustomize/README.md#additional-deployment-instructions-memorystore)** section below to properly undo the Terraform changes.
+
+## Additional Deployment Instructions (Memorystore)
```

Comment:
Suggestion: Since this `Additional Deployment Instructions (Memorystore)` section is largely about Terraform, I would move it all to [/terraform/README.md](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/terraform). Moreover, I would:
1. remove the `Run the infrastructure changes with Terraform` from `kustomize/README.md` altogether
2. assume that the user has already `git clone`d this repo inside `terraform/README.md` (i.e., remove step 1 and change step 2 to `cd terraform`).
3. Add a step or note inside `terraform/README.md`, below the `gcp_project_id` step that says something like, "If you are using a Memorystore Redis cache, change the value of `memorystore = false` to `memorystore = true`."